tests/unit/readers/test_reader_postgres.py

```
@@ -243,36 +229,18 @@ def test_missing_connection_field_raises_runtime_error(
 
         with (
             patch("boto3.Session") as mock_session,
-            pytest.raises(RuntimeError, match="host, user, password"),
+            pytest.raises(RuntimeError, match="PostgreSQL configuration error.*Missing PostgreSQL secret fields"),
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)